pkgs/applications/office/mendeley/default.nix

```diff
@@ -1,6 +1,7 @@
 { fetchurl, stdenv, dpkg, makeWrapper, which
-,gcc, liborc, xorg, qt4, zlib
-, ...}:
+, alsaLib, dbus, expat, fontconfig, freetype, gcc, glib, libcap
+, liborc, libxml2, libxslt, nss, nspr, sqlite, xorg, qt55, zlib
```

Comment:
You should use `qt55.callPackage` in `all-packages.nix` instead of importing `qt55` into the expression. This ensures that everything which depends on Qt 5 is built with the same version.